.maintain/frame-weight-template.hbs

```diff
@@ -45,10 +45,6 @@ pub trait WeightInfo {
     ) -> Weight;
     {{/each}}
     {{#if (eq pallet "pallet_gear")}}
```

Comment:
mb we can also remove `{{#if (eq pallet "pallet_gear")}} .. {{/if}}` and other usages with `pallet_gear`